src/lib/core/public-api.ts

```diff
@@ -18,5 +18,3 @@ export * from './label/label-options';
 export * from './ripple/index';
 export * from './selection/index';
 
-// TODO: don't have this
-export * from './testing/month-constants';
```

Comment:
People might be importing from here, you need to re-export it:

```ts
export {JAN, FEB, ...} from '@angular/cdk/testing';
```

(only re-export the ones that are currently exported, not all of `@angular/cdk/testing`, and leave the comment to eventually remove)

Reply:
It looks like you didn't make this change?

Reply:
We talked about how testing is not exposed externally and our solution was to put `BREAKING CHANGES:` in the description

Reply:
We talked about removing this, but clarified that we should still have this anyway